templates/taxi/car_form.html

```diff
@@ -1,7 +1,11 @@
 {% extends "base.html" %}
 {% load crispy_forms_filters %}
 {% block content %}
-   <h1>{{ object|yesno:"Update,Create" }} Car</h1>
```

Comment:
this was correct and now it isn't. You should change manufacturer form, not car form